evm/src/cpu/cpu_stark.rs

```diff
@@ -22,6 +22,12 @@ use crate::memory::{NUM_CHANNELS, VALUE_LIMBS};
 use crate::stark::Stark;
 use crate::vars::{StarkEvaluationTargets, StarkEvaluationVars};
 
+const EXTRA_OPS: [usize; 3] = [
+    COL_MAP.op.syscall,
+    COL_MAP.op.exception,
```

Comment:
Does this branch need a rebase? I thought these have been combined into one flag.

Reply:
The duplicated constraints have been merged (cf #1146), but the flags haven't been combined yet AFAIK.